www/docs/server/procedures.md

```diff
@@ -5,336 +5,88 @@ sidebar_label: Define Procedures
 slug: /server/procedures
 ---
 
-Procedures in tRPC are very flexible primitives to create backend functions; they use a builder pattern which means you can create reusable base procedures for different parts of your backend application.
+In tRPC a Procedure is ultimately just a function which you want to expose to your client. A Procedure can be either:
+
+- a `Query` - used to fetch data, generally does not change any data
+- a `Mutation` - used to send data, often for create/update/delete purposes
+
+Procedures in tRPC are very flexible primitives to create backend functions, they use an immutable builder pattern which means you can [create reusable base procedures](#reusable-base-procedures) which share functionality among multiple procedures.
 
 :::tip
 
-- A procedure can be viewed as the equivalent of a REST-endpoint or a function.
+- A procedure can be viewed as the equivalent of a REST-endpoint, and under the hood are built on HTTP, but are more focused towards the needs of application, rather than representing "resources".
 - There's no internal difference between queries and mutations apart from semantics.
-- Defining publicProcedure is the same for queries, mutations, and subscription with the exception that subscriptions need to return an `observable` instance.
+- Defining a publicProcedure is the same for queries, mutations, and subscriptions, with the exception that subscriptions need to return an `observable` instance.
```

Comment:
- `Defining a publicProcedure`... should maybe just be a procedure (without public)?
- is this bullet point needed at all? maybe mentioning subscriptions in the first set of bullet points further up is enough, and then all other information about subscriptions can be in the subscriptions part of the docs

Reply:
I agree, I want to minimise references to subscriptions here, and that 3rd point I'm not sure what it's trying to say that's not clear from usage, and what's left makes a tidy paragraph. Though the need for tip at all is debatable